styles.css

```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+}
+
+img {
+  height: 8rem;
+}
+
+.grass {
+  background-color: rgb(40, 141, 37);
```

Comment:
If you just put the background color on the `body`, it will fill the full viewport without needing a size